doc/manual/getting_started_mgmt.rst

```diff
@@ -46,6 +46,30 @@ The dashboard should display the list of experiments from the repository folder
 
 .. note:: Multiple clients may be connected at the same time, possibly on different machines, and will be synchronized. See the ``-s`` option of ``artiq_dashboard`` and the ``--bind`` option of ``artiq_master`` to use the network. Both IPv4 and IPv6 are supported.
 
+(Optional) Using a monitor/injection proxy for core device
```

Comment:
Seems not optional when using with dashboard.
Just make sure the doc clearly states that you need to start the proxy. Should just be a doc issue if it is intended.
@sbourdeauducq